qb/select_test.go

```diff
@@ -28,6 +28,11 @@ func TestSelectBuilder(t *testing.T) {
 			B: Select("cycling.cyclist_name").Columns("id", "user_uuid", "firstname"),
 			S: "SELECT id,user_uuid,firstname FROM cycling.cyclist_name ",
 		},
+		// Add a SELECT AS column.
```

Comment:
Please follow the convention and do not put a dot at the end here. It's not a sentence anyway.

Reply:
Done